integration/container/run_test.go

```diff
@@ -0,0 +1,65 @@
+package container
+
+import (
+	"context"
+	"testing"
+
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/integration/util"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/docker/docker/pkg/parsers/kernel"
+)
+
+func daemonStorageDriver(ctx context.Context, cli client.APIClient) (string, error) {
+	info, err := cli.Info(ctx)
+	if err != nil {
+		return "", err
+	}
+	return info.Driver, nil
+}
+
+// #34573
+func TestRunVolumeLeakage(t *testing.T) {
+	defer setupTest(t)()
+	client := request.NewAPIClient(t)
+
+	// Note whether the test is running on devicemapper (where the error being
+	// tested ocurred). We run it anyway just to be safe.
+	driver, _ := daemonStorageDriver(context.Background(), client)
+	if driver != "devicemapper" {
+		t.Logf("Active driver (%s) is not devicemapper, continuing anyway.", driver)
```

Comment:
I don't think this is the right way to test storage drivers.  These "integration" tests are very slow, we need to be conservative about the test we add here (see [TESTING](https://github.com/moby/moby/blob/master/TESTING.md)).

We should be able to test storage drivers without the daemon running, which makes it much easier and faster to test them. It looks like each graphdriver has a test suite which uses `graphtest`. We should test this case in that suite.

Why is it not possible to just check that the mount point doesn't exist anymore from that test suite?